site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -255,12 +260,18 @@ endif::includeRoles[]
 [field]#application.samlv2Configuration.audience# [type]#[String]# [optional]#Optional# [default]#defaults to `issuer`# [since]#Available since 1.6.0#::
 The audience for the SAML response sent to back to the service provider from FusionAuth. Some service providers require different audience values than the `issuer` and this configuration option lets you change the `audience` in the response.
 
-[field]#application.samlv2Configuration.callbackURL# [type]#[String]# [required]#Required# [since]#Available since 1.6.0#::
+[field]#application.samlv2Configuration.authorizedRedirectURLs# [type]#[Array<String>]# [optional]#Optional# [since]#Available since 1.20.0#::
```

Comment:
At least one value is required. 
https://github.com/FusionAuth/fusionauth-api/blob/8453df081bbd31be6f166305acddd662a0fe17f0/src/main/java/io/fusionauth/api/service/application/DefaultApplicationService.java#L729